<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>788</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>613</width>
    <height>788</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>youtubeIcon.png</normaloff>youtubeIcon.png</iconset>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>631</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
	background-color: rgb(0, 0, 0);
	border: 2px solid rgb(255, 255, 255); 
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: none;
	font: 57 30pt &quot;YouTube Sans Dark&quot;;
	color: rgb(255, 255, 255);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>YouTube</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: none;
	border: none;
	image: url(:/icons/youtubeIcon.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter your email account</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border-bottom: 2px solid rgb(255, 255, 255);
	border-right: none;
	border-left: none;
	border-top: none;
	background-color: transparent;
	font: 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
}

QLineEdit:focus {
	border-bottom: 3px solid rgb(255, 255, 255);
	border-right: none;
	border-left: none;
	border-top: none;
}

QToolTip {
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: none;
	font: 57 20pt &quot;YouTube Sans Dark&quot;;
	color: rgb(255, 255, 255);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: none;
	font: 57 20pt &quot;YouTube Sans Dark&quot;;
	color: rgb(255, 255, 255);
	border: none;
}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter your password account</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border-bottom: 2px solid rgb(255, 255, 255);
	border-right: none;
	border-left: none;
	border-top: none;
	background-color: transparent;
	font: 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
}

QLineEdit:focus {
	border-bottom: 3px solid rgb(255, 255, 255);
	border-right: none;
	border-left: none;
	border-top: none;
}

QToolTip {
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="revealPasswordButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>370</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Reveal Password</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: transparent;
	
	image: url(:/icons/opennedEyeeIcon.png);
}

QToolTip {
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="LogButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>331</width>
      <height>81</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Log In to Youtube</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(206, 0, 0);
	color: rgb(255, 255, 255);
	font: 20pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 10px;
}

QPushButton:hover {
	background-color: rgb(255, 0, 0);
}

QPushButton:pressed {
	background-color: rgb(206, 0, 0);
}

QToolTip {
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createAccountButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>560</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Sign Up</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: transparent;
	color: rgb(255, 255, 255);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton:hover {
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	text-decoration: underline;
}

QPushButton:pressed{
	text-decoration: none;
}

QToolTip {
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Create Account</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>760</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: none;
	color: rgb(255, 255, 255);
	background-color: none;
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>Â© YouTube is a trademark of Google LLC. Not affiliated.
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="youtubeButton">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>55</y>
      <width>301</width>
      <height>55</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>610</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: none;
	border: none;
	color: rgb(255, 0, 0);
	font: 10pt &quot;MS Shell Dlg 2&quot;;
}</string>
    </property>
    <property name="text">
     <string>Error message</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>785</y>
     <width>611</width>
     <height>4</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
	background-color: transparent;
	border: none;
}

QProgressBar:chunk {
	background-color: rgb(255, 0, 0);
}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
